cypress/integration/persianBodySpec.js

```diff
@@ -45,6 +41,9 @@ describe('Article Body Tests', () => {
   });
 
   it('should render a title', () => {
-    renderedTitle('پهپادی که برایتان قهوه می‌آورد - BBC News فارسی');
+    cy.window().then(win => {
+      const { seoHeadline } = win.SIMORGH_DATA.data.promo.headlines;
+      renderedTitle(`${seoHeadline} - BBC News فارسی`);
```

Comment:
In a future PR, we can have the site name in a config file, so that BBC News / BBC News فارسی can be pulled from that.

Reply:
Agreed. I tried to do this but it didn't seem to work. Something to look at for the future.

Reply:
Hmmm it would make CodeClimate (and me!) happy if we could refactor out the duplication, by exporting another `const` in `bodyTestHelper.js`.

```js
const { seoHeadline } = getHeadline(win.SIMORGH_DATA);
```

...and in bodyTestHelper.js:

```js
export const getHeadline = simorghData => {
  return simorghData.data.promo.headlines;
};
```